CHANGELOG.md

```diff
@@ -9,6 +9,51 @@ The ManageIQ organization is continuously adding new smaller repositories.  The
 - [manageiq-providers-vmware CHANGELOG](https://github.com/ManageIQ/manageiq-providers-vmware/blob/master/CHANGELOG.md)
 - [manageiq-ui-classic CHANGELOG](https://github.com/ManageIQ/manageiq-ui-classic/blob/master/CHANGELOG.md)
 
+## Unreleased - as of Sprint 67 ending 2017-08-21
+
+### [Added](https://github.com/ManageIQ/manageiq/issues?utf8=%E2%9C%93&q=milestone%3A%22Sprint%2067%20Ending%20Aug%2021%2C%202017%22%20label%3Aenhancement%20)
+- Don't use secure sessions in containers [(#15819)](https://github.com/ManageIQ/manageiq/pull/15819)
+- Add metrics default limit to API settings [(#15797)](https://github.com/ManageIQ/manageiq/pull/15797)
+- Missing settings for a cloud batch saving and adding shared_options [(#15792)](https://github.com/ManageIQ/manageiq/pull/15792)
+- Get tag details for no specific model [(#15788)](https://github.com/ManageIQ/manageiq/pull/15788)
+- Add group in manageiq payload for ansible automation. [(#15787)](https://github.com/ManageIQ/manageiq/pull/15787)
+- Validate that an entitlement has one kind of managed filter [(#15786)](https://github.com/ManageIQ/manageiq/pull/15786)
+- Batch saving timestamps enhancement [(#15774)](https://github.com/ManageIQ/manageiq/pull/15774)
```

Comment:
Things like this need context or need to be removed entirely.  A user will have no idea what this means.